artifacts/testing/C01-device-errors.feature

```diff
@@ -0,0 +1,118 @@
+Feature: CAMARA Common Artifact C01 - Test scenarios for device errors
+
+    Common error scenarios for POST operations with device as input either in the request
+    body or implied from the access.
+
+    Artifact parameters (to be replaced by values according to the API operation):
+    
+    - {{feature_identifier}} 
+
+    This is not a complete feature but a collection of scenarios that can be applied
+    with minor modifications to test plans.
+
+    These scenarios assume that other properties not explicitly mentioned in the scenario
+    are set by default to a valid value. This can be specified in the feature Background.
+
+    # Error scenarios for management of input parameter device
+
+    # If the access token identifies a device, error 422 UNNECESSARY_DEVICE may be returned instead
+    @{{feature_identifier}}_C01.01_device_empty
+    Scenario: The device value is an empty object
+        Given the header "Authorization" is set to a valid access which does not identifiy a single device
+        And the request body property "$.device" is set to: {}
+        When the HTTP "POST" request is sent
+        Then the response status code is 400
+        And the response property "$.status" is 400
+        And the response property "$.code" is "INVALID_ARGUMENT"
+        And the response property "$.message" contains a user friendly text
+
+
+    # If the access token identifies a device, error 422 UNNECESSARY_DEVICE may be returned instead
+    @{{feature_identifier}}_C01.02_device_identifiers_not_schema_compliant
+    Scenario Outline: Some device identifier value does not comply with the schema
+        Given the header "Authorization" is set to a valid access which does not identifiy a single device
+        And the request body property "<device_identifier>" does not comply with the OAS schema at "<oas_spec_schema>"
+        When the HTTP "POST" request is sent
+        Then the response status code is 400
+        And the response property "$.status" is 400
+        And the response property "$.code" is "INVALID_ARGUMENT"
+        And the response property "$.message" contains a user friendly text
+        
+        Examples:
+            | device_identifier          | oas_spec_schema                             |
+            | $.device.phoneNumber       | /components/schemas/PhoneNumber             |
+            | $.device.ipv4Address       | /components/schemas/NetworkAccessIdentifier |

```

Comment:
Mismatch of $.device.ipv4Address with value as /components/schemas/NetworkAccessIdentifier .Similarly for $.device.ipv6Address  & $.device.networkIdentifier parameters as well.

Reply:
Fixed, thanks!